spec/helper-spec.js

```diff
@@ -2,7 +2,7 @@
 
 import * as fs from 'fs'
 import * as path from 'path'
-const helpers = require('../lib/helpers')
+const helpers = require('../src/helpers')
```

Comment:
Just checked the relevant commit, CI should be building before running the specs.